haskell_scotty/DBConfig.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module DBConfig (DBConfig(..), readDBConfig)
```

Comment:
Without the (..) you only get the structure name itself. With (..) you also get the structure internals. It's a way of doing information hiding or not.